.kuzzlerc

```diff
@@ -118,9 +118,11 @@
       "controllers": {
         "auth": {
           "actions": {
-            "login": true,
+            "logout": true,
```

Comment:
That doesn't seem logical. The problem here is that if you attempt a `logout` action (or any action actually) with an expired token, you shouldn't get an `UnauthorizedError` but a new `TokenExpiredError` instead.